manifests/p/pizzaboxer/Bloxstrap/2.7.0/pizzaboxer.Bloxstrap.locale.en-US.yaml

```
@@ -0,0 +1,53 @@
+# Created with WinGet Releaser v2 using Komac v1.11.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: pizzaboxer.Bloxstrap
+PackageVersion: 2.7.0
+PackageLocale: en-US
+Publisher: pizzaboxer
+PublisherUrl: https://pizzaboxer.xyz
+PublisherSupportUrl: https://github.com/pizzaboxer/bloxstrap/issues
+PackageName: Bloxstrap
+PackageUrl: https://github.com/pizzaboxer/bloxstrap

```

Point to the new website:
```suggestion
PackageUrl: https://bloxstrap.pizzaboxer.xyz/
```

---

Oh thanks, didn't catch that.

---

@pizzaboxer offtopic, but can you give me contributor role on the Bloxstrap server lol

---

@sitiom Didn't know you were in the Discord server, what's your username?

---

Same username: @sitiom